storage/src/vespa/storage/persistence/persistencethread.cpp

```diff
@@ -92,6 +92,11 @@ class ResultTaskOperationDone : public spi::OperationComplete {
     vespalib::ISequencedTaskExecutor::ExecutorId   _executorId;
 };
 
+vespalib::string
+createThreadName(size_t stripeId) {
+    return vespalib::make_string("Thread %zu", stripeId);
```

Comment:
Consider naming it "Persistence Thread x"

Reply:
Fixed in https://github.com/vespa-engine/vespa/pull/14920